src/app/tabs/debugger/debuggerUI/vmDebugger/MemoryPanel.js

```diff
@@ -15,7 +15,7 @@ MemoryPanel.prototype.update = function (calldata) {
 }
 
 MemoryPanel.prototype.render = function () {
-  return yo`<div id="memorypanel" class="text-monospace small">${this.basicPanel.render()}</div>`
```

Comment:
is the panel well formatted when removing `text-monospace` ? I remenber using font `monospace` for this specific reason?

Reply:
![debug_2](https://user-images.githubusercontent.com/30843294/72513926-c8d4b880-3873-11ea-9e6d-c1b71658edbe.png)

Tabs looks like this